interpreter.py

```diff
@@ -22,6 +22,7 @@ def I(l,i=None,p=0,m=[0],o="",r=0,de=0,_=None):
    g=[t:=t+(Oi(k)==8+d)-(Oi(k)==9+d)for k in l].index(0)
    while m[p] and f:i,p,m,o=I(l[:g+1],i,p,m,o,r+1,de,_); f=n==8
    l=l[g:]
+  elif n in[12,13]: m[p]=sorted([0,m[p]+1-[12,0,13].index(n),_W(-1)])[1]
```

Comment:
When the flag `-w 0` is set, the cells do not wrap and can hold arbitrary negative and positive values. For that setting, I guess `±` should always increment `1` and for `∓` we can still decide.

I was thinking that it could always decrement, because negative values are allowed, or `∓` could wrap to `0` if the value of the cell would drop below `0`. The "always decrement" behaviour isn't that interesting, because people can just use `-` for that.

Reply:
I like the prevention of decrementing past zero.

Reply:
Yes, me too. With the current implementation, if a cell has a negative value and is then decremented with `∓` it becomes `0`. I kind of like this behaviour; what do you think?

Reply:
Oh hmm, I didn't even realize. I guess it's one of those situations where it truly is both a bug and a feature. If someone instead wants to keep decrementing, I suppose they can just use the `-` operator.